tpgtools/handwritten/orgpolicy_utils.go

```diff
@@ -0,0 +1,26 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceOrgPolicyPolicyCustomImport(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	if err := parseImportId([]string{
```

Comment:
I would add a comment which says "this differs from the normal import method because it permits slashes in the first element and requires the parent to start at the beginning of the string.

Reply:
Done

Reply:
I can't think of any off the top of my head, but do we see this import pattern in other resources? Any merit to a generic solution?

Reply:
There are other resources in the DCL that allow a forward slashes in the values for certain fields. Usually, as it is here, this is because the field is a parent field and there can be more than one type of parent for the resource.